pystiebeleltron/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.0.1"
```

Comment:
I think this needs to be "0.1.1". (Somehow this comment didn't make it into my last review)

Reply:
Changed to 0.1.1